examples/nextjs-whiteboard-advanced/src/index.tsx

```diff
@@ -291,9 +291,14 @@ function Canvas() {
     [canvasState, selection]
   );
 
-  const unselectLayers = useMutation(({ setMyPresence }) => {
-    setMyPresence({ selection: [] }, { addToHistory: true });
-  }, []);
+  const unselectLayers = useMutation(
+    ({ setMyPresence }) => {
+      if (selection.length > 0) {
+        setMyPresence({ selection: [] }, { addToHistory: true });
+      }
+    },
+    [selection]
+  );
```

Comment:
Unrelated to this bug, but FYI today I'll work on exposing the current presence from within the `useMutation` hook, so we can remove the dependency on `selection` here for this, which will make this mutation a lot simpler to express. (See https://github.com/liveblocks/liveblocks/pull/384#discussion_r967249977)